auth/server/src/main/java/com/seagates3/policy/PolicyValidator.java

```diff
@@ -68,7 +77,7 @@ ServerResponse validateEffect(Effect effectValue) {
       if (Statement.Effect.valueOf(effectValue.name()) == null) {
         response = responseGenerator.malformedPolicy("Invalid effect : " +
                                                      effectValue.name());
-        LOGGER.error("Effect value is invalid in bucket policy");
+        LOGGER.error("Effect value is invalid in policy");
```

Comment:
print policy name

Reply:
Policy name is not available here. 
But I have added the debug log with policy name included at the time of calling validation from PolicyController#Create.